### PR TITLE
TLUNIZIN-875 Mask Course ID in API Calls

### PR DESCRIPTION
**Pull request by dovek**
Due: 1/29
@dlhaines 
@gsilver

### CODE REVIEW

**Comment by gsilver** [comment]
LGTM!

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 1]
```diff
@@ -708,7 +710,13 @@ private boolean isAllowedApiRequest(HttpServletRequest request) {
 			isAllowedRequest=isApiFoundIntheList(url);
 		}
 
+		//typical crosslist call is not allowed in the LTI version
```
Comment (dlhaines) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: Can we chat about isAllowedApiRequest?

File: src/main/webapp/assets-lti/js/controllers/controllers.js [hunk 2]
```diff
@@ -156,7 +156,8 @@ canvasSupportApp.controller('courseController', ['Course', 'Courses', 'Sections'
     // for each added section call a factory that will do a post
     $scope.course.xLists =[];
     _.each(addedSections, function(section){
-      var xListUrl = 'manager/api/v1/sections/' + section.id + '/crosslist/' + courseId;
+      //var xListUrl = 'manager/api/v1/sections/' + section.id + '/crosslist/' + courseId;
```
Comment (dlhaines) [inline src/main/webapp/assets-lti/js/controllers/controllers.js]: Why keep this commented line?

**Comment by dlhaines** [comment]
Looks good.

**Pull request merged**